### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by MayurChawla**
Added Lists
- Ordered Lists
- Unordered Lists
- Nested Lists
- Stacked Lists

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice work

File: Components/Lists/Lists.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    --light-color : #ffeff5;
+    --dark-color : #72022d;
+
+    --badge-primary : #0d6efd;
+    --badge-success : #198754;
+    --badge-warning : #ffc107;
+    --badge-link : #fd7e14;
+    --badge-info : #0dcaf0;
+    --badge-inside-button : goldenrod;
+}
```
Comment (ZuberDunge) [inline Components/Lists/Lists.css:15]: You can add spacing after each {}.

File: Components/Lists/Lists.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+* {
```
Comment (ZuberDunge) [inline Components/Lists/Lists.css:16]: You can move this to reset.css

File: Components/Lists/Lists.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    width: 0px;
+}
+html {
+    width: 100%;
+    height: 100%;
+}
+body {
+    background-color: #F7F7F7;
+    width: 100%;
+    display: flex;
```
Comment (ZuberDunge) [inline Components/Lists/Lists.css:31]: avoid giving styling to body tag, use some class, otherwise it'll create problems for the user.

**Review by ZuberDunge — COMMENTED** [review]
Yuu can shift resetting CSS and root variables to a separate file, so you don't have to declare in every files, and also avoid giving styling to tags, specially body tag. Added few suggestions, apart from that, Everything looks good to me.

File: Components/Lists/Lists.css (hunk 1 above)
Comment (MayurChawla) [reply Components/Lists/Lists.css:15]: yes noted

File: Components/Lists/Lists.css (hunk 2 above)
Comment (MayurChawla) [reply Components/Lists/Lists.css:16]: Will do that post finalizing components

**Pull request merged**